### ISSUE TITLE
Aria roles attributes on fine position button

### ISSUE THREAD

**Issue by dascritch** [dascritch/cpu-audio#188]
Actual fine position buttons doesn't have aria-* attributes. Check if we need them (someone must test with a braille tablet, and with a vocal assistant)
